### ISSUE TITLE
All inputs of type text must have a label

### ISSUE THREAD

**Issue by messman** [htmllint/htmllint#74]
This label should be connected with an id or wrapping the input. 

Reference: [CalPoly 508 Standards](http://warc.calpoly.edu/accessibility/508indepth/forms.html)
